openstack/datadog_checks/openstack/openstack.py

```diff
@@ -1176,6 +1176,7 @@ def ensure_auth_scope(self, instance):
 
         return instance_scope
 
+    @trace_func
```

Comment:
Is there a way to add it to all checks without to add this decorator here?

Reply:
I don't see an easy was to do that as, since we don't use a lot of frameworks that the tracing client supports out of the box, we need to do this "manual instrumentation". I think having a single decorator per check isn't a bad trade off here though. I also don't think we necessarily want/need it for all checks right now 🙂

Reply:
Yeah! That's fair. Thanks